src/config/dynamic-theme-fixes.config

```diff
@@ -9752,6 +9752,8 @@ span[class^="ts-"]
 img[src$="Rename_icon_Cyr.svg.png"]
 img[alt="Aksara Sunda.png"]
 img[alt="Aksara kawi name.png"]
+img[src*="/static/images/mobile/copyright/wikipedia-wordmark"]
+img[src*="/static/images/mobile/copyright/wikipedia-tagline"]
```

Comment:
Where can I find this element?

Reply:
https://wikipedia.org/static/images/mobile/copyright/wikipedia-wordmark-fr.svg
https://wikipedia.org/static/images/mobile/copyright/wikipedia-tagline-pt.svg
It's these two elements. For me only the portuguese version of Wikipedia has the new design. These two elemets were not inverted before I applied this fix. I made it a substring since the language indicator at the end varies.

Reply:
Doesn't seems to harm or break anything so good to be merged